docs/source/usage/iss/iss_pipeline.py

```diff
@@ -0,0 +1,64 @@
+import starfish
+from starfish.image import Filter, Registration, Segmentation
+from starfish.spots import SpotFinder, TargetAssignment
+from starfish.types import Indices
+
+
+def iss_pipeline(fov, codebook):
+    primary_image = fov.primary_image
+
+    # register the raw images
+    registration = Registration.FourierShiftRegistration(
+        upsampling=1000,
+        reference_stack=fov['dots']
+    )
+    registered = registration.run(primary_image, in_place=False)
+
+    # filter raw data
+    masking_radius = 15
+    filt = Filter.WhiteTophat(masking_radius, is_volume=False)
+    filtered = filt.run(registered, verbose=True, in_place=False)
+
+    # detect spots using laplacian of gaussians approach
+    p = SpotFinder.GaussianSpotDetector(
+        min_sigma=1,
+        max_sigma=10,
+        num_sigma=30,
+        threshold=0.01,
+        measurement_type='mean',
+    )
+    blobs_image = fov['dots'].max_proj(Indices.ROUND, Indices.Z)
+    intensities = p.run(filtered, blobs_image=blobs_image)
+
+    # decode the pixel traces using the codebook
+    decoded = codebook.decode_per_round_max(intensities)
+
+    # segment cells
+    seg = Segmentation.Watershed(
+        dapi_threshold=.16,
+        input_threshold=.22,
+        min_distance=57,
+    )
+    regions = seg.run(primary_image, fov['nuclei'])
+
+    # assign spots to cells
+    ta = TargetAssignment.PointInPoly2D()
+    assigned = ta.run(decoded, regions)
+
+    return assigned, regions
+
+
+# process all the fields of view, not just one
+def process_experiment(experiment: starfish.Experiment):
+    decoded_intensities = {}
+    regions = {}
+    for name_, fov in experiment.items():
```

Comment:
`name` is not a special kw.  would just rename it `name`.  `fov_name` would be even better.